index.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const app = express()
+
+
+app.use(express.static('public'))
+app.set('view engine', 'pug')
+app.get('/hunters', (request, response) =>
+{
+  response.render('index')
+})
```

Comment:
You should have something at `/`, a user won't know to go to `/hunters` if they've never been to your site before